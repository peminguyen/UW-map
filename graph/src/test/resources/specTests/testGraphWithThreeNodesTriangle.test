CreateGraph graphWithThreeNodesTriangle

AddNode graphWithThreeNodesTriangle A
AddNode graphWithThreeNodesTriangle B
AddNode graphWithThreeNodesTriangle C

AddEdge graphWithThreeNodesTriangle A B eAB
AddEdge graphWithThreeNodesTriangle B C eBC
AddEdge graphWithThreeNodesTriangle C A eCA

ListNodes graphWithThreeNodesTriangle
ListChildren graphWithThreeNodesTriangle A
ListChildren graphWithThreeNodesTriangle B
ListChildren graphWithThreeNodesTriangle C