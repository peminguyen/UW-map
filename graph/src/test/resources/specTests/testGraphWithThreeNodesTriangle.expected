created graph graphWithThreeNodesTriangle

added node A to graphWithThreeNodesTriangle
added node B to graphWithThreeNodesTriangle
added node C to graphWithThreeNodesTriangle

added edge eAB from A to B in graphWithThreeNodesTriangle
added edge eBC from B to C in graphWithThreeNodesTriangle
added edge eCA from C to A in graphWithThreeNodesTriangle

graphWithThreeNodesTriangle contains: A B C
the children of A in graphWithThreeNodesTriangle are: B(eAB)
the children of B in graphWithThreeNodesTriangle are: C(eBC)
the children of C in graphWithThreeNodesTriangle are: A(eCA)
